scripts2/findWord.js

```diff
@@ -0,0 +1,16 @@
+const getWordlePage = async (browserInstance) => {
+  const page = await browserInstance.newPage();
+  await page.goto("https://www.wordle.net/");
+  return page;
+};
+
+const findWord = async (browserInstance, wordList) => {
+  const page = getWordlePage(browserInstance);
+  let word = "";
+
+  // Figure out word using wordList
+
+  return word;
+};
```

Comment:
Make sure to split findWord function into a modular process. Some modules of findWord can be:

- Navigate to page
- Try word
- Get next word that may satisfy requirements